### PR TITLE
Steps cleanup and rancher restore update

### PR DESCRIPTION
**Pull request by theabmitra**

### CODE REVIEW

**Review by kminder — APPROVED** [review]
LGTM

File: content/en/docs/uninstall/backup/backup.md [hunk 1]
```diff
@@ -121,9 +121,18 @@ $ kubectl apply -f - <<EOF
 EOF
 ```
 
-
 The operator creates the backup file, in the `*.tar.gz` format, and stores it in the location configured in the `storageLocation` field.
 
+When backup is complete, then rancher backup operator creates a file on th S3 Compatible Object store.
```
Comment (rosemarymarano) [inline content/en/docs/uninstall/backup/backup.md:126]: re-write sentence: When the backup is complete, then the rancher-backup operator creates a file on the S3 Compatible Object store.

Also, please on line 77, change the heading to ## Rancher backup

File: content/en/docs/uninstall/backup/restore.md [hunk 2]
```diff
@@ -46,20 +46,14 @@ To initiate a MySQL restore, from an existing backup, you need to recreate the M
...
-5. Clean up MySQL pods and PVC from the system.
+4. Clean up MySQL pods and PVC from the system.
 
     ```shell
     $ helm delete mysql -n keycloak
     $ kubectl delete pvc -n keycloak -l tier=mysql
     ```
 
-7. Trigger a MySQL restore by executing the Helm chart as follows.
+5. Trigger a MySQL restore by executing the Helm chart as follows.
```
Comment (rosemarymarano) [inline content/en/docs/uninstall/backup/restore.md:56]: executing -> (Avoid when you mean performing a programming instruction. Use run or process instead.) -> running

**Review by rosemarymarano — COMMENTED** [review]
Edits and comments. I'll re-review wen you are done.

File: content/en/docs/uninstall/backup/restore.md (hunk 2 above)
Comment (theabmitra) [reply content/en/docs/uninstall/backup/restore.md:56]: ok, this dosent make sense. executing, invoking and running are all synonyms in this context .

File: content/en/docs/uninstall/backup/restore.md [hunk 3]
```diff
@@ -154,8 +136,10 @@ Restoring rancher is done by creating a custom resource that indicates to `ranch
    - Custom Resource Definitions (CRDs)
    - Cluster-scoped resources
    - Namespace resources
+  
+   *NOTE* The `backupFilename` is retrieved from the rancher backups created earlier.
```
[the remark this answers is not in the document]
Comment (theabmitra) [reply content/en/docs/uninstall/backup/restore.md]: done

File: content/en/docs/uninstall/backup/restore.md [hunk 4]
```diff
@@ -87,15 +81,15 @@ To initiate a MySQL restore, from an existing backup, you need to recreate the M
             --values mysql-values.yaml
     ```   
 
-8. Wait for the MySQL cluster to be online after performing the restore command. Ensure that the `STATUS` is `ONLINE` and the count under `ONLINE` matches the `INSTANCES`.
+6. Wait for the MySQL cluster to be online after performing the restore command. Ensure that the `STATUS` is `ONLINE` and the count under `ONLINE` matches the `INSTANCES`.
```
[the remark this answers is not in the document]
Comment (theabmitra) [reply content/en/docs/uninstall/backup/restore.md]: done

File: content/en/docs/uninstall/backup/backup.md [hunk 5]
```diff
@@ -121,9 +121,18 @@ $ kubectl apply -f - <<EOF
 EOF
 ```
 
-
 The operator creates the backup file, in the `*.tar.gz` format, and stores it in the location configured in the `storageLocation` field.
 
+When backup is complete, then rancher backup operator creates a file on th S3 Compatible Object store. 
+
+The filename can be retrieved as shown below:
```
[the remark this answers is not in the document]
Comment (theabmitra) [reply content/en/docs/uninstall/backup/backup.md]: done

File: content/en/docs/uninstall/backup/backup.md (hunk 1 above)
Comment (theabmitra) [reply content/en/docs/uninstall/backup/backup.md]: done

File: content/en/docs/uninstall/backup/restore.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/uninstall/backup/restore.md]: This is not fixed. 
Let me fix this.

[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/uninstall/backup/restore.md]: Fixed now.

**Review by rosemarymarano — APPROVED** [review]
LGTM

**Review by MelwynPaul — APPROVED** [review]
Looks good.

**Pull request merged**